decompiler/util/default.json

```diff
@@ -220,11 +220,33 @@
       "description": "Transform while-loops to for-loops although the resulting for-loop would be empty.",
       "is_hidden_from_gui": false
     },
-    "rename_for_loop_variables": {
-      "default": true,
-      "type": "boolean",
+    "rename_for_loop_variables" : {
```

Comment:
Short question, the previous "false" would now the empty array? If so, then it should be specified somewhere, or we should have both options.

+  Perhaps also rename option to "for_loop_variable_names"?

Reply:
Ok to the name change and yes, the empty list would be "false".